junifer/datagrabber/__init__.py

```diff
@@ -10,4 +10,3 @@
 from .multiple import MultipleDataGrabber
 from .pattern import PatternDataGrabber
 from .pattern_datalad import PatternDataladDataGrabber
-from .aomic1000 import DataladAOMIC1000
```

Comment:
now you need to `from .aomic import DataladAOMICID1000, DataladAOMICPIOP1, DataladAOMICPIOP2`